Customer Demo/Customer Demo/Screens/SignIn/SignInViewController.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  SignInViewController.swift
+//  Customer Demo
+//
+//  Copyright 2022 Ably Real-time Ltd (ably.com)
+//
+
+import Foundation
+import UIKit
+import Network
+
+class SignInViewController: UIViewController, ActivityIndicating {
+    
+    @IBOutlet private var usernameTextField: UITextField!
+    @IBOutlet private var passwordTextField: UITextField!
+    @IBOutlet private var submitButton: UIButton!
+    
+    let presenter = SignInPresenter()
+    
+    override func viewDidLoad() {
+        presenter.viewController = self
+        
+        super.viewDidLoad()
+        
+        submitButton.addTarget(self, action: #selector(submitButtonTapped), for: .touchUpInside)
```

Comment:
Out of curiosity (not a change request) - why didn't you handle it using the `@IBAction`?

Reply:
it's clearer for me and easier to find out how it works